### ISSUE TITLE
TypeError: a bytes-like object is required, not 'str'

### ISSUE THREAD

**Issue by klonuo** [pyinstaller/pyinstaller#2014]
I pip installed pyinstaller, then tried it on simple script, where I got several exceptions on `_getImports_pe()`:

``` pytb
...
4570 WARNING: Can not get binary dependencies for file: D:\Programs\MiKTeX 2.9\miktex\bin\x64\api-ms-win-crt-stdio-l1-1-0.dll
Traceback (most recent call last):
  File "c:\python35\lib\site-packages\PyInstaller\depend\bindepend.py", line 697, in getImports
    return _getImports_pe(pth)
  File "c:\python35\lib\site-packages\PyInstaller\depend\bindepend.py", line 124, in _getImports_pe
    dll, _ = sym.forwarder.split('.')
TypeError: a bytes-like object is required, not 'str'
...
```

So it seems that `sym.forwarder` can be of type `bytes` while it is expected to be of type string, in which case something like this should handle it:

```diff
--- C:\Python35\Lib\site-packages\PyInstaller\depend\bindepend.py
+++ (clipboard)
@@ -119,7 +119,9 @@
         for sym in exportSymbols.symbols:
             if sym.forwarder is not None:
                 # sym.forwarder is for example 'KERNEL32.EnterCriticalSection'
-                dll, _ = sym.forwarder.split('.')
+                sym_forwarder = (isinstance(sym.forwarder, bytes) and
+                                 sym.forwarder.decode() or sym.forwarder)
+                dll, _ = sym_forwarder.split('.')
                 dlls.add(winutils.convert_dll_name_to_str(dll) + ".dll")
```

**Comment by stephenrauch** [comment]
Fixed by #1981

**Comment by htgoebel** [comment]
@stephenrauch Thanks for spotting the duplicate.
